api/type.rb

```diff
@@ -16,12 +16,14 @@
 
 module Api
   # Represents a property type
+  # rubocop:disable Metrics/ClassLength
   class Type < Api::Object::Named
     # The list of properties (attr_reader) that can be overridden in
     # <provider>.yaml.
     module Fields
       include Api::Object::Named::Properties
 
+      attr_reader :default_value
```

Comment:
nit: it is common to have a unit associated with the value. like for example a timeout you could do: timeout_ms=long, or instead timeout=TimeUnit (where the type holds both the number and unit)